docs/usage/configuration-options.md

```diff
@@ -631,6 +631,11 @@ Because `fileMatch` is mergeable, you don't need to duplicate the defaults and c
 If you configure `fileMatch` then it must be within a manager object (e.g. `dockerfile` in the above example).
 The full list of supported managers can be found [here](https://docs.renovatebot.com/modules/manager/).
 
+## filterUnavailableUsers
+
+When this option is enabled PRs are not assigned to users that are unavailable.
+Please note that this will only have an effect when the platform supports the concept of user availabity (currently supported by GitLab).
+
```

Comment:
@HonkingGoose Would be great to get some feedback on the description and the name of the option. 🙏

Reply:
I like the general idea!

How  do you like `filterOutUnavailableUsers` or some variation on that?
Maybe saying that this option "filters out"  something is easier to understand than just "filter something"?
But maybe the word filter is enough to imply filtering out things?

Reply:
Sounds cool to me. I had "filter out" in mind first, but it sounded to German (my mother tongue 😀) to me, so I shortened it to just "filter"